save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/SettingsView.kt

```diff
@@ -34,8 +34,6 @@ val cardHeight: CSSProperties = jso {
 }
 
 val userSettingsView: FC<SettingsProps> = FC { props ->
-    // This is needed for us to wait for userInfo uploaded
-    val (loadedUserInfo, setLoadedUserInfo) = useStateFromProps(props.userInfo ?: UserInfo(""))
```

Comment:
here @sanyavertolet . Previously it was:

App.kt -> BasicRouting { userInfo } -> userSettingsView { userInfo } -> userSettingsView { useStateFromProps(userInfo) } -> RightCol { userInfo }